src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js

```diff
@@ -0,0 +1,362 @@
+    browser.waitForElementVisible(localGroupPopup);
+    localGroupPopup.changeDescription(browser.globals.test_string_300.slice(0,256));
+    localGroupPopup.clickHeader();
+    browser.assert.containsText('//div[contains(@class, "v-snack__content")]', 'Request failed with status code 500');;
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")] and .//*[contains(text(), "Group4")]]')
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+    localGroupPopup.changeDescription('GroupChanged');
+    localGroupPopup.clickHeader();
+    browser.assert.containsText('//div[contains(@class, "v-snack__content")]', 'Description saved');
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")] and .//*[contains(text(), "GroupChanged")]]')
+    browser.end();
+
+  },
+  'Security server delete local group': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const clientLocalGroups = clientInfo.section.localGroups;
+    const localGroupPopup = mainPage.section.localGroupPopup;
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Open TestGov Internal Servers and local group details view
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    // Delete and cancel
+    localGroupPopup.deleteThisGroup();
+    browser.waitForElementVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');
+    localGroupPopup.cancelMemberRemove();
+    browser.waitForElementNotVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');   
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")]]')
+
+    // Delete and confirm
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    localGroupPopup.deleteThisGroup();
+    browser.waitForElementVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');
+    localGroupPopup.confirmMemberRemove();  
```

Comment:
This confirms _local group removal_, not _local group member removal_, right? Should the name be different?

Reply:
Implemented